_config.yml

```diff
@@ -64,7 +64,12 @@ navigation:
         position: 20
 
 ## List custom TOC order for folders here
-
+     "*components/form/formitems":
+        title: "Form Items"
+        position: 5
+     "*components/form/layout":
```

Comment:
Consider moving high in the ToC, maybe as a third item right after the validation - the easy appearance settings would be some of the first things people can look for and try.

Reply:
done